v2/cn/ann-1019003001.md.tt2

```diff
@@ -28,6 +35,8 @@ A brife overview of the most noteable changes:
   recreate request buffer in balancer phase.
 * feature: supported receiveany on `ngx.req.socket(true?)` socks.
 * feature: implemented the new `ngx.ssl.server_port()` API to get server port.
+* Detect SSE4.2 support dynamically.
+* bugfix: ngx.flush never blocks on http v2.
```

Comment:
We should use the past tense for bug that no longer exists and we just state the new behavior in highlight, not state the bad incorrect behavior:

> The `ngx.flush(true)` Lua API can now work properly for HTTP 2.0 requests.